sds/src/secondary_validation/sweden_pin_checksum.rs

```diff
@@ -0,0 +1,72 @@
+pub struct SwedenPINChecksum;
+use crate::secondary_validation::Validator;
+
+const MULTIPLIERS: [u32; 9] = [2, 1, 2, 1, 2, 1, 2, 1, 2];
+
+fn sum_of_digits(digits: u32) -> u32 {
+    let mut sum = 0;
+    let mut num = digits;
+    while num > 0 {
+        sum += num % 10;
+        num /= 10;
+    }
+    sum
+}
```

Comment:
This will be useful for a few other checksums, I will move it to `mod.rs` where we already have a few other helper methods 👍